tests/_RandomGenerator.py

```diff
@@ -199,3 +199,19 @@ def int64_tuple(self, size: int, low: int = -1 << 63, high: int = 1 << 63) -> tu
 
     def ascii(self, size: int) -> str:
         return ''.join([chr(self.uint8(32, 127)) for _ in range(size)])
+
+    def rand_like(self, x: torch.Tensor) -> torch.Tensor:
+        """Generates a tensor with the same shape as x filled with uniform random numbers."""
```

Comment:
uniform random would be symmetric around 0. Maybe mention "in the intervall [0, 1)" or something like this